docs/docs/dev/networking/dns.md

```diff
@@ -223,3 +161,10 @@ Kuma DNS allocates a VIP for every service within a mesh. Then, it creates an ou
      }
     },
 ```
+
+::: tip
+The following setup will work when `serviceVipEnabled=true` which is a default value.
+
+The preferred way to define hostnames is using [Virtual Outbounds](../policies/virtual-outbound).
+Virtual Outbounds also makes it possible to define dynamic hostnames using specific tags or to expose services on a different port.
```

Comment:
Might be worth mentioning that serviceVipEnabled will be set to false in the future?

Reply:
I'd rather hold off with this until we are 100% sure we want to get rid of `serviceVipEnabled`.